### ISSUE TITLE
What does isRegisterLocationUpdate means?

### ISSUE THREAD

**Issue by dacianf** [rekab-app/background_locator#85]
@mehdok 
I've seen that in ios isRegisterLocationUpdate returns the value of initialized variable, which is never set on false. Shouldn't it be set on false in unRegisterLocationUpdate or how do we really unregister the location service?
Beside this, I've also got a duplicate register plugin key after a start - stop - start which I've solved with a dispach_once on `registerPlugins(_headlessRunner);` I don't know if this is a good approach.

**Comment by fnicastri** [comment]
Noticed the same, in my code I have a patch (based on an old version) to uniformate the behavior of isRegisterLocationUpdate on both platforms.
@mehdok are you interested in a pr?

**Comment by mehdok** [comment]
@fnicastri Sure, I would appreciate any help. But since this is an old issue, I'm not sure if we already fixed it or not @dacianf . I can remember that **duplicate register plugin key** is fixed;

**Comment by fnicastri** [comment]
> @fnicastri Sure, I would appreciate any help. But since this is an old issue, I'm not sure if we already fixed it or not @dacianf . I can remember that **duplicate register plugin key** is fixed;

@mehdok 

referring to the isRegisterLocationUpdate different behavior on iOS and Android.
On Android it is true only when the locator is running, on iOS it is true when the locator is initialized, not just when you are using it.

**Comment by dacianf** [comment]
@mehdok Yes, the duplicate register plugin key is fixed, but as @fnicastri said, isRegisterLocationUpdate has different meanings on each platform.
Maybe we could add two different methods, isPluginRegisterd and isServiceRunning. What do you think about this @mehdok @fnicastri ?

**Comment by fnicastri** [comment]
> @mehdok Yes, the duplicate register plugin key is fixed, but as @fnicastri said, isRegisterLocationUpdate has different meanings on each platform.
> Maybe we could add two different methods, isPluginRegisterd and isServiceRunning. What do you think about this @mehdok @fnicastri ?

Add isServiceRunning would be good!
